autorag/nodes/passagereranker/koreranker.py

```diff
@@ -0,0 +1,89 @@
+import asyncio
+from typing import List, Tuple
+
+import numpy as np
+import torch
+from transformers import AutoModelForSequenceClassification, AutoTokenizer
+
+from autorag.nodes.passagereranker.base import passage_reranker_node
+
+
+@passage_reranker_node
+def koreranker(queries: List[str], contents_list: List[List[str]],
+               scores_list: List[List[float]], ids_list: List[List[str]],
+               top_k: int) -> Tuple[List[List[str]], List[List[str]], List[List[float]]]:
+    """
+    Rerank a list of contents based on their relevance to a query using ko-reranker.
+    ko-reranker is a reranker based on korean (https://huggingface.co/Dongjin-kr/ko-reranker).
+    :param queries: The list of queries to use for reranking
```

Comment:
Press enter between function description and parameter description.
(It is rule of sphinx-apidoc)